book/src/developers/contributing/build_instructions/linux.md

```diff
@@ -71,12 +71,12 @@ ENR entry for IP blank. Some users report better connections over VPN."
 ### Optional flags for conflicting nodes
 
 The discovery and JSON-RPC ports may conflict with an existing an Ethereum client
-on the same machine.
+or other software on the same machine.
 
-`--discovery-port <port>`. If an Ethereum consensus client is already running, it may be using
-the default port 9000.
+`--discovery-port <port>`. The default port is 9009. Pick something else if in use.
 
-`--web3-http-address <ip_address>:<port>`. If an Ethereum execution client is already running, it may be using the default port 8545. The localhost IP address (127.0.0.1) is recommended here.
+`--web3-http-address <ip_address>:<port>`. If an Ethereum execution client is already running,
+it may be using the default port 8545. The localhost IP address (127.0.0.1) is recommended here.
```

Comment:
Hmmm... so if execution client's default port is `8545` (eg [geth](https://geth.ethereum.org/docs/interacting-with-geth/rpc)). Then doesn't it make sense to also change our default listening port so there isn't a conflict there? For basically the same reason we're changing the discovery port (though this change would definitely be out of scope for this pr, but if we think the change is worthwhile, let's open an issue to track it)